components/pd_client/src/client.rs

```diff
@@ -325,7 +325,9 @@ impl PdClient for RpcClient {
                 .rl()
                 .client_stub
                 .get_region_by_id_async_opt(&req, Self::call_option())
-                .unwrap();
+                .unwrap_or_else(|e| {
+                    panic!("fail to request PD {} err {:?}", "get_region_by_id", e)
```

Comment:
Why not inline "get_region_by_id"?

Reply:
I think they are the same.

Reply:
I don't think they are the same. When searching for error, I normally paste the error message to quickly navigate to the location. Current format makes it impossible to search using "get_region_by_id err".